plugins/modules/katello_content_view_filter.py

```diff
@@ -172,9 +172,10 @@
 
 RETURN = ''' # '''
 
-from ansible.module_utils.foreman_helper import KatelloAnsibleModule
+from ansible.module_utils.foreman_helper import KatelloAnsibleModule, _foreman_spec_helper
 
 content_filter_spec = {
+    'id': {},
```

Comment:
shouldn't these be `invisible`?

Reply:
Maybe if they were used in the `foreman_spec` that gives raise to the `argument_spec`, but they aren't.